### PR TITLE
Removing add_msvc_runtime_flag.

### PR DESCRIPTION
**Pull request by ayermolo**
Removing add_msvc_runtime_flag. Not clear it is still necessary, and interfered in creating VS Solutions that support multiple configurations.
Came out of: https://github.com/onnx/onnx/pull/1966

### CODE REVIEW

File: cmake/Utils.cmake [hunk 1]
```diff
@@ -1,18 +1,12 @@
 #
-# Add MSVC RunTime Flag
+# Add MSVC RunTime Flag, this part is necessary for tests in CI
```
Comment (houseroad) [inline cmake/Utils.cmake]: Is it only required by CI? or real world case also need it?

CI should just be one instance of real world case in my opinion.

**Comment by houseroad** [comment]
To be honest, I still didn't see a strong motivation to drop this options...

Could you further elaborate?

**Comment by linkerzhang** [comment]
@houseroad  this PR is based on the discussion of #1966, I believe.

File: cmake/Utils.cmake (hunk 1 above)
Comment (ayermolo) [reply cmake/Utils.cmake]: Well I added in case few down the road same question is raised.

**Comment by ayermolo** [comment]
Any way to re-kick CI? Doesn't look like failure is related. "No output has been received in the last 10m0s, this potentially indicates a stalled build or something wrong with the build itself."

**Comment by ayermolo** [comment]
@houseroad great, thanks!

**Comment by ayermolo** [comment]
@zrphercule All good on your side?

**Comment by ayermolo** [comment]
Awesome, thanks!

**Pull request merged**